### PR TITLE
[BUILD][STF] Move STF

### PR DESCRIPTION
**Pull request by jafey**
I moved the STF-Parts (Framework, Tests, Self-Tests) to new packages dpp.stf and dpp.stf.test.
@m273d15 added the gradle part, so that you can build the project with gradle.

### CODE REVIEW

File: de.fu_berlin.inf.dpp.stf.test/sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,16 @@
+sonar.projectName        = Saros/Eclipse
```
Comment (m273d15) [inline de.fu_berlin.inf.dpp.stf.test/sonar-project.properties]: this file is obsolete. The sonar properties are maintained in the build.gradle (e.g. [here](https://github.com/saros-project/saros/blob/master/de.fu_berlin.inf.dpp.core/build.gradle)).
Add the following block to the build.gradle file of the project:
```gradle
sonarqube {
  properties {
    property 'sonar.projectName', <ADD NAME HERE>
    property 'sonar.projectDescription', <ADD DESCRIPTION HERE>
    property 'sonar.projectVersion', <ADD VERSION HERE>
  }
}
```

Comment (m273d15) [inline de.fu_berlin.inf.dpp.stf.test/sonar-project.properties]: You have to use a name that is unique in the project `Saros/Eclipse` is already used.

File: de.fu_berlin.inf.dpp.stf.test/sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+sonar.projectDescription = Saros STF Test
+sonar.projectVersion     = 14.11.28.DEVEL
```
Comment (m273d15) [inline de.fu_berlin.inf.dpp.stf.test/sonar-project.properties]: Use an internal version as `0.1.0`

File: de.fu_berlin.inf.dpp.stf/sonar-project.properties [hunk 3]
```diff
@@ -1,14 +1,18 @@
 sonar.projectName        = Saros/STF
 sonar.projectDescription = Saros Test Framework
-sonar.projectVersion     = 0.1.0.DEVEL
+sonar.projectVersion     = 14.11.28.DEVEL
```
Comment (m273d15) [inline de.fu_berlin.inf.dpp.stf/sonar-project.properties]: Please use an internal version as `0.1.0`. I see no reason to increase the version of the STF for each release.

File: de.fu_berlin.inf.dpp.stf/sonar-project.properties [hunk 4]
```diff
@@ -1,14 +1,18 @@
 sonar.projectName        = Saros/STF
```
Comment (m273d15) [inline de.fu_berlin.inf.dpp.stf/sonar-project.properties]: This file is not needed anymore. See my comment in `de.fu_berlin.inf.dpp.stf.test/sonar-project.properties`

File: de.fu_berlin.inf.dpp/test/stf/de/fu_berlin/inf/dpp/stf/test/consistency/EditDuringNonHostAddsProjectTest.java [hunk 5]
```diff
@@ -1,99 +0,0 @@
-package de.fu_berlin.inf.dpp.stf.test.consistency;
```
Comment (m273d15) [inline de.fu_berlin.inf.dpp/test/stf/de/fu_berlin/inf/dpp/stf/test/consistency/EditDuringNonHostAddsProjectTest.java]: why do you remove this test?

File: .gitignore [hunk 6]
```diff
@@ -1,6 +1,7 @@
 bin/
 bin_test/
 /.idea/*.iws
+/.idea/saros.iml
```
[the remark this answers is not in the document]
Comment (jafey) [reply .gitignore]: I don't now why intellij created this file for me. But actually it isn't there so I simply remove the line from .gitignore

File: de.fu_berlin.inf.dpp/test/stf/de/fu_berlin/inf/dpp/stf/test/consistency/EditDuringNonHostAddsProjectTest.java (hunk 5 above)
Comment (jafey) [reply de.fu_berlin.inf.dpp/test/stf/de/fu_berlin/inf/dpp/stf/test/consistency/EditDuringNonHostAddsProjectTest.java]: I think something went wrong during moving the files. I found a empty EditDuringNonHostAddsProjectTest.java at the old location. I will restore the file and move it to the new location

**Comment by m273d15** [comment]
You still have to rebase this PR properly. The whole directory naming has changed. Therefore you receive a lot of merge conflicts.

**Comment by jafey** [comment]
I tryed to rebase but it is to hard for me 😞 I think it will be easier to do the same refactoring again based on the current master.

**Comment by jafey** [comment]
See #489

**Pull request closed without merging**